easybuild/easyconfigs/g/grpcio/grpcio-1.57.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'grpcio'
+version = '1.57.0'
+
+homepage = 'https://grpc.io/'
+description = """gRPC is a modern, open source, high-performance remote procedure call (RPC)
+framework that can run anywhere. gRPC enables client and server applications to
+communicate transparently, and simplifies the building of connected systems."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+use_pip = True
+sanity_pip_check = True
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('RE2', '2023-08-01'),
+    ('Abseil', '20230125.3'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('protobuf-python', '4.24.0'),
+]
+
+exts_list = [
+    (name, version, {
+        'modulename': 'grpc',
+        'preinstallopts': (
+            "export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=%(parallel)s && "
```

Comment:
Please add
```
"sed -i 's@/usr@%(sysroot)s/usr@g' setup.py && " &&
```
to this. It replaces all hardcoded `/usr` instances in the `setup.py` of GRPC by the value configured for the `sysroot` EasyBuild configuration item (`/usr` by default, but could be a different path on systems with a custom `sysroot`. See the `--sysroot` option for EasyBuild on https://docs.easybuild.io/version-specific/eb-help/#override-options).

Note that the previous EasyConfig also had this, see [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/19f73c78317c94f0d9291148e1c265244aa152d4/easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.13.0-foss-2023a.eb#L84)

Reply:
Thanks, I missed that, putting it back.